doc/classes/X509Certificate.xml

```diff
@@ -17,12 +17,25 @@
 				Loads a certificate from [param path] ("*.crt" file).
 			</description>
 		</method>
+		<method name="load_from_string">
+			<return type="int" enum="Error" />
+			<param index="0" name="str_value" type="String" />
+			<description>
+				Loads a certificate from [param str_value] (String value).
```

Comment:
The wording from `Saves a certificate to a String value which is returned.` is difference and is missing an article. "a"